acquisitions/src/main/resources/thunderjet/mod-orders/features/independent-acquisitions-unit-for-ordering-and-receiving.feature

```diff
@@ -0,0 +1,720 @@
+    And request
+    """
+    {
+      "title": "Sample Title",
+      "poLineId": "#(poLineId1)",
+      "instanceId": "f1b57aeb-23c9-4386-bcb8-efda56878267",
+      "contributors": [
+        {
+          "contributor": "Ed Mashburn",
+          "contributorNameTypeId": "fbdd42a8-e47d-4694-b448-cc571d1b44c3"
+        }
+      ],
+      "publisher": "Schiffer Publishing",
+      "publishedDate": "1972",
+      "edition": "Third Edt.",
+      "subscriptionFrom": "2018-10-09T00:00:00.000Z",
+      "subscriptionInterval": 824,
+      "subscriptionTo": "2020-10-09T00:00:00.000Z",
+      "claimingActive": false,
+      "claimingInterval": 0,
+      "isAcknowledged": true,
+      "productIds": [
+        {
+          "productId": "9780764354113",
+          "productIdType": "8261054f-be78-422d-bd51-4ed9f33c3422",
+          "qualifier": "(paperback)"
+        }
+      ],
+    }
+    """
+    When method POST
+    Then status 403
+
+  Scenario: Make acqUnit2 as active and repeat step again - operation should be allowed
+    # 1. Make acqUnit2 active
+    * configure headers = headersAdmin
+    Given path 'acquisitions-units/units', acqUnitId2
+    And request
+    """
+    {
+      "id": '#(acqUnitId2)',
+      "protectUpdate": true,
+      "protectCreate": true,
+      "protectDelete": true,
+      "protectRead": true,
+      "isDeleted": false,
+      "name": "testAcqUnit2"
+    }
+    """
+    When method PUT
+    Then status 204
+
+    # 2. Create acq unit membership
+    Given path 'acquisitions-units/memberships'
+    And request
+    """
+      {
+        "id": '#(acqUnitMembershipId5)',
+        "userId": "00000000-1111-5555-9999-999999999992",
+        "acquisitionsUnitId": "#(acqUnitId2)"
+      }
+    """
+    When method POST
+    Then status 201
+
+    # 2. POST Title
+    Given path 'orders/titles'
+    And request
+    """
+    {
+      "title": "Sample Title",
+      "poLineId": "#(poLineId1)",
+      "instanceId": "f1b57aeb-23c9-4386-bcb8-efda56878267",
+      "contributors": [
+        {
+          "contributor": "Ed Mashburn",
+          "contributorNameTypeId": "fbdd42a8-e47d-4694-b448-cc571d1b44c3"
+        }
+      ],
+      "publisher": "Schiffer Publishing",
+      "publishedDate": "1972",
+      "edition": "Third Edt.",
+      "subscriptionFrom": "2018-10-09T00:00:00.000Z",
+      "subscriptionInterval": 824,
+      "subscriptionTo": "2020-10-09T00:00:00.000Z",
+      "claimingActive": false,
+      "claimingInterval": 0,
+      "isAcknowledged": true,
+      "productIds": [
+        {
+          "productId": "9780764354113",
+          "productIdType": "8261054f-be78-422d-bd51-4ed9f33c3422",
+          "qualifier": "(paperback)"
+        }
+      ],
+      "acqUnitIds": ["#(acqUnitId2)"]
+    }
+    """
+    When method POST
+    Then status 201
+
+  ## Receive/Edit piece checks
+  Scenario: Verify Edit piece functionality
+    # Assign acqUnit1 to Order, open and receive this Order. Assign acqUnit2 to related Title.
```

Comment:
comment and code should match, in comment we have acqUnit2, in code - acqUnit3

Reply:
Fixed